.devcontainer/docker-compose-vscode.yml

```diff
@@ -22,6 +22,7 @@ services:
       - TIME_SYNC_TOPIC=time_sync
       - TIME_SYNC_PORT=7575
       - SIM_V2X_PORT=5757
+      - SIM_INTERACTION_PORT=7576
```

Comment:
Not sure I understand how you split these two PRs up. To me the easiest way to split them up would have been one handles the new registration and the other handles the forwarding. Not sure why the SIM Interaction Port would be added here.

Reply:
This PR was up first to test send the detected object to this port.